tests/geom/elem_test.C

```diff
@@ -480,8 +478,17 @@ public:
         CPPUNIT_ASSERT(parent->has_children());
         CPPUNIT_ASSERT(!parent->has_ancestor_children());
         CPPUNIT_ASSERT(!parent->interior_parent());
-        CPPUNIT_ASSERT_EQUAL(parent, elem->top_parent());
-        CPPUNIT_ASSERT_EQUAL(parent, parent->top_parent());
+        if (n == 1)
```

Comment:
Hi Roy, a bit too late I know. :/
If you want, I think you could write this whole 11-line block as:

```
CPPUNIT_ASSERT_EQUAL(elem->top_parent(), parent->top_parent());
CPPUNIT_ASSERT((n == 1) != (parent != elem->top_parent()));
```

or if that's too cumbersome,

```
CPPUNIT_ASSERT_EQUAL(elem->top_parent(), parent->top_parent());
if (n == 1)
  CPPUNIT_ASSERT_EQUAL(parent, elem->top_parent());
else
  CPPUNIT_ASSERT(parent != elem->top_parent());
```

Reply:
I like that second one.  Not enough to open a new PR for it, though.  :-)